### PR TITLE
Fix java_api_test CI

### PR DESCRIPTION
**Pull request by huayuan4396**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily receiving feedbacks. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
3. If the modification has a dependency on downstream projects of a newer version, this PR should be tested with all supported versions of downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: .github/workflows/java_api.yml [hunk 1]
```diff
@@ -17,6 +17,35 @@ concurrency:
 jobs:
   test_java_api:
     runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        task_config:
+          - task: ImageClassification
+            configs: 'https://media.githubusercontent.com/media/hanrui1sensetime/mmdeploy-javaapi-testdata/master/resnet.tar'
```
Comment (RunningLeon) [inline .github/workflows/java_api.yml:24]: ci still fails

Comment (RunningLeon) [inline .github/workflows/java_api.yml:24]: the matrix will create too many parallel jobs. Maybe this is not a good way.

**Comment by huayuan4396** [comment]
Now the workflow is ok but the some tasks fail.

**Comment by huayuan4396** [comment]
The build.xml is modified to make sure the workflow would fail if any java test fails.

File: .github/workflows/java_api.yml [hunk 2]
```diff
@@ -75,5 +75,150 @@ jobs:
...
+          java_command+=" $GITHUB_WORKSPACE/$resource_file"
+          echo "java_command: $java_command"
+          cd $GITHUB_WORKSPACE/demo/java
+          ant -DtaskName=$task -DjarDir=$GITHUB_WORKSPACE/opencv/build/bin -DlibDir=${OPENCV_DIR}/build/lib:$GITHUB_WORKSPACE/build/lib -Dcommand="$java_command"
+
+      - name: Run ImageSegmentation Test
+        if: always()
+        run: |
+          task="ImageSegmentation"
+          model_url="https://media.githubusercontent.com/media/hanrui1sensetime/mmdeploy-javaapi-testdata/master/fcn.tar"
```
Comment (RunningLeon) [inline .github/workflows/java_api.yml:116]: can you fix the failures of some tasks?

**Comment by RunningLeon** [comment]
@huayuan4396 any update on this PR?

**Pull request closed without merging**